src/auth/auth.controller.ts

```diff
@@ -0,0 +1,19 @@
+import { Controller, Post, Body } from '@nestjs/common';
+import { AuthService } from './auth.service';
+import { HttpCode } from '@nestjs/common/decorators';
+
+@Controller('auth')
+export class AuthController {
+  constructor(private authService: AuthService) {}
+
+  @Post('signup')
+  async signUp(@Body('token') oAuthToken: string): Promise<unknown> {
+    return await this.authService.signUp(oAuthToken);
+  }
+
+  @HttpCode(200)
+  @Post('signin')
+  async signin(@Body('token') oAuthToken: string): Promise<unknown> {
+    return await this.authService.signUp(oAuthToken);
```

Comment:
Should be signIn instead of signUp?

```suggestion
    return await this.authService.signIn(oAuthToken);